codebase/gatling-tests/src/main/scala/com/virtuslab/auctionHouse/perfTests/AuctionHouseSimulation.scala

```diff
@@ -0,0 +1,54 @@
+package com.virtuslab.auctionHouse.perfTests
```

Comment:
Can't we just leave 1 empty class extending AuctionHouseSimulation in test and remove duplicated classes?

Reply:
I think we can.